Scala/Scala.sublime-syntax

```diff
@@ -140,12 +213,25 @@ contexts:
             - match: '\)'
               pop: true
             - include: pattern-match
-        - match: '=>|⇒|\b(if)\b'
+        - match: '\b(if)\b'
+          captures:
+            1: keyword.control.flow.scala
+          set:
+            - match: '=>|⇒'
```

Comment:
Needs a scope, possibly a `keyword` variant (I realized this may not be 100% correct according to scala semantics, but I think it would make sense to most users)